string name
string[] errors
string type	# IGNORE to filter this name of component, OBSERVE to actively sense it